nntrainer/include/model_loader.h

```diff
@@ -94,6 +98,20 @@ class ModelLoader {
                             NeuralNetwork &model,
                             const std::string &backbone_name);
 
+  /**
+   * @brief     load backbone config from ini
+   * @param[in] ini dictionary containing the config
```

Comment:
please update the brief. And I wonder that what external means.

Reply:
done.